templates/shared_services/sonatype-nexus/Dockerfile.tmpl

```diff
@@ -0,0 +1,21 @@
+FROM debian:stretch-slim
+
+ARG BUNDLE_DIR
+
+RUN apt-get update && apt-get install -y ca-certificates
+
+# This is a template Dockerfile for the bundle's invocation image
```

Comment:
comments needed?

Reply:
don't know - it's the boilerplate comments, I don't seen any reason to remove it?

Reply:
I think they do no harm and might provide some instruction to the next person who wants to work with the dockerfile?